src/cdb.js

```diff
@@ -1,36 +1,32 @@
 // Creates cdb object, mutated in the entry file cartodb.js
 // Used to avoid circular dependencies
-var cdb = window.cdb = window.cartodb;
+var cdb = {};
 
 // Might be set in different bundles (e.g. cartodb.js vs. torque), avoid overriding it once set
-if (!cdb) {
-  cdb = window.cdb = window.cartodb = {};
-  if (!window.JST) window.JST = {};
```

Comment:
Not used here anymore, so :fire: completely, turned out to be a dependency for `cdb.init` in v3. From v4 this should live in the editor, added a note on the issue for that there :point_right: https://github.com/CartoDB/cartodb/issues/6068